docs/content/features/exploiters/mssql.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "MSSQL Exploiter"
+draft: false
+description: "Exploits MSSQL by taking advantage of insecure configuration"
+tags: ["exploiter", "mssql", "brute force"]
+pre: "<i class='fa fa-database'></i> "
+---
+
+## MSSQL
+
+[MSSQL (Microsoft SQL Server)](
+https://learn.microsoft.com/en-us/sql/sql-server/what-is-sql-server) is a
+relational database management system (RDBMS) developed by Microsoft whose
+functions include managing, storing, retrieving, manipulating, and analyzing
+data efficiently.
+
+[`xp_cmdshell`](
+https://learn.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/xp-cmdshell-transact-sql)
+is a procedure in MSSQL that allows users to execute Windows shell commands
+from within the SQL Server environment.
+
+## Exploitation
+
+Machines with MSSQL that have `xp_cmdshell` enabled may be accessible to
+attackers if they come across the correct credentials.
+
+Infection Monkey's MSSQL exploiter uses brute-force to propagate to a victim
+by taking advantage of insecure MSSQL configuration. It leverages the
+`xp_cmdshell` feature to execute commands on the server.
```

Comment:
I feel like these two paragraphs are basically building up to say the same thing. Maybe they can be combined.

Reply:
I'd prefer keeping them separate. This is how it is in all the exploiter pages. The first paragraph says something about the service's vulnerability and the second explains how it works in Infection Monkey.

Reply:
Maybe, but most of them aren't paragraphs, they're standalone sentences.

Reply:
Combining them in all pages